bin/metaBackend.py

```diff
@@ -21,47 +21,506 @@
+                str(ownerId))
+        db.execCommand0(cmd, opts)
+        repoId = db.execCommand1("SELECT LAST_INSERT_ID()")[0]
+        cmd = "INSERT INTO DbMeta(dbMetaId, dbName, connHost, connPort) "
+        cmd += "VALUES(%s,%s,%s,%s)"
+        db.execCommand0(cmd, (str(repoId), dbName, host, str(port)))
+
+        for t in theTable:
+            cmd = 'INSERT INTO DDT_Table(dbMetaId, tableName, descr) '
+            cmd += 'VALUES(%s, %s, %s)'
+            db.execCommand0(cmd, (str(repoId), t,
+                                  theTable[t].get("description", "")))
+            tableId = db.execCommand1("SELECT LAST_INSERT_ID()")[0]
+            isFirst = True
+            for c in theTable[t]["columns"]:
+                if isFirst:
+                    cmd = 'INSERT INTO DDT_Column(columnName, tableId, '
+                    cmd += 'ordinalPosition, descr, ucd, units) VALUES '
+                    opts = ()
+                    isFirst = False
+                else:
+                    cmd += ', '
+                cmd += '(%s, %s, %s, %s, %s, %s)'
+                opts += (c["name"], str(tableId), str(c["ord_pos"]),
+                         c.get("description", ""), c.get("ucd", ""),
+                         c.get("unit", ""))
+            db.execCommand0(cmd, opts)
+
+    def addUser(self, muName, fName, lName, affil, email):
+        """
+        Add user.
+
+        @param muName MySQL user name
+        @param fName  first name
+        @param lName  last name
+        @param affil  short name of the affilliation (home institution)
+        @param email  email address
+        """
+        db = Db(read_default_file=self._msMysqlAuthF)
+        cmd = "SELECT instId FROM Institution WHERE instName = %s"
+        instId = db.execCommand1(cmd, (affil,))
+        if instId is None:
+            raise MetaBException(MetaBException.INST_NOT_FOUND, affil)
+        cmd = "INSERT INTO User(mysqlUserName, firstName, lastName, email, instId) "
+        cmd += "VALUES(%s, %s, %s, %s, %s)"
+        db.execCommand0(cmd, (muName, fName, lName, email, str(instId[0])))
+
+    def addInstitution(self, name):
+        """
+        Add institution.
+
+        @param name  the name
+        """
+        db = Db(read_default_file=self._msMysqlAuthF)
+        cmd = "INSERT INTO Institution(instName) VALUES(%s)"
+        db.execCommand0(cmd, (name,))
+
+    def addProject(self, name):
+        """
+        Add project.
+
+        @param name  the name
+        """
+        db = Db(read_default_file=self._msMysqlAuthF)
+        cmd = "INSERT INTO Project(projectName) VALUES(%s)"
+        db.execCommand0(cmd, (name,))
+
+####################################################################################
+####################################################################################
+####################################################################################
+class CommandParser(object):
```

Comment:
It looks like this is reusing a lot of code from qserv's `qserv-admin.py` (or maybe other script that I missed). WE should not repeat the same thing again and again. Also we need better/simpler parsing if we are going to continue writing scripts like this.